sources/platform/actors/development/actor_definition/output_schema/index.md

```diff
@@ -0,0 +1,296 @@
+---
+title: Actor output schema
+sidebar_label: Actor output schema
+sidebar_position: 5
+description: Learn how to define and present output of your Actor.
+slug: /actors/development/actor-definition/output-schema
+---
+
+**Learn how to define and present the output of your Actor.**
+
+---
+
+The Actor output schema builds upon the schemas for the dataset and key-value store, and defines where the output of an Actor run is stored. It allows you to define templates for URLs where users can find the output generated by your Actor. The outputs defined in the schema are used in the UI to manage how the output is displayed and are returned in the Actor run's `GET` endpoint for automated systems using the API.
+
+## Example
+
+Consider a very simple example Actor that calls `Actor.setValue()` to save two files into the key-value store:
+
+```javascript title="main.js"
+import { Actor } from 'apify';
+// Initialize the JavaScript SDK
+await Actor.init();
+
+/**
+ * Store data in key-value store
+ */
+await Actor.setValue('document-1.txt', 'my text data', { contentType: 'text/plain' });
+
+await Actor.setValue(`image-1.jpeg`, imageBuffer, { contentType: 'image/jpeg' });
+
+// Exit successfully
+await Actor.exit();
+```
+
+To let users know that the output is stored in the key-value store, you can update the `.actor/actor.json` configuration like this:
+
+```json title=".actor/actor.json"
+{
+    "actorSpecification": 1,
+    "name": "Actor Name",
+    "title": "Actor Title",
+    "version": "1.0.0",
+    "output": {
+        "actorOutputSchemaVersion": 1,
+        "title": "Output schema of the Actor",
+        "properties": {
+            "files": {
+                "type": "string",
+                "title": "Files",
+                "template": "{{links.apiDefaultKeyValueStoreUrl}}/keys"
+            }
+        }
+    }
+}
+```
+
+The schema above defines one output called `files` which uses the `template` to define a template which links to the key-value store `GET keys` API endpoint of the default key-value store.
```

Comment:
To be honest, this seems very difficult for me to understand. 😃 I am not sure, if it can be explained more clearly, though.